toolkit/scripts/get_config_deps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# $1 - CONFIG_FILE
+# $2 - CONFIG_BASE_DIR
+
+if [[ $# -ne 2 ]]
+then
+    echo "ERROR: Must provide CONFIG_FILE and CONFIG_BASE_DIR to get_config_deps.sh" >&2
+    exit 1
+fi
+
+CONFIG_FILE="$1"
+CONFIG_BASE_DIR="$2"
+
+config_other_files=$(grep -E '.json|.sh' $CONFIG_FILE| sed 's/\"Path\"\://' | tr ", \n" " " | tr "\"" " " | xargs)
```

Comment:
If you want to go the optional `jq` route, I tried this again and as far as I can tell it does work in a script (wonder if it was just an issue with make..., wouldn't surprise me).

I like using `jq` since its more resilient than grep/sed, but I think making it optional for now isn't a bad idea.

```suggestion
if jq --version &> /dev/null
then
    config_other_files="$(jq -r '.SystemConfigs[]?.PackageLists[]' "${config_file}")"
    config_other_files="${config_other_files_relative} $(jq -r '.SystemConfigs[]?.PostInstallScripts[]?.Path' "${config_file}")"
else
    config_other_files=$(grep -E '.json|.sh' $CONFIG_FILE| sed 's/\"Path\"\://' | tr ", \n" " " | tr "\"" " " | xargs)
fi
```

Reply:
Offline: won't bother with jq, not worth the complexity